products/sqlite.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+alternate_urls:
+-   /sqlite3
+versionCommand: sqlite3 --version
+releasePolicyLink: https://www.sqlite.org/support.html
+changelogTemplate: "https://www.sqlite.org/changes.html#version_{{'__LATEST__'|replace:'.','_'}}"
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Support Status
+
+# This git mirror only contains versions from 3.6.10.
+auto:
+-   git: https://github.com/sqlite/sqlite.git
+    regex: '^version-(?<major>\d+)\.(?<minor>\d+)(\.(?<patch>\d+))?$'
+
+releases:
+-   releaseCycle: "3"
+    eol: false
+    latest: "3.40.1"
+    latestReleaseDate: 2022-12-28
+    releaseDate: 2004-06-18
+
+-   releaseCycle: "2"
+    releaseDate: 2001-09-28
```

Comment:
@marcwrobel isnt these page also need a bit  Normalization ?

Reply:
Why ? It looks already normalized to me.